app/src/main/java/org/p2p/wallet/swap/ui/orca/OrcaSwapFragment.kt

```diff
@@ -149,6 +127,51 @@ class OrcaSwapFragment :
         presenter.loadInitialData()
     }
 
+    private fun setupViews() = with(binding) {
+        toolbar.setNavigationOnClickListener { presenter.onBackPressed() }
+        toolbar.setOnMenuItemClickListener { menu ->
+            if (menu.itemId == R.id.settingsMenuItem) {
+                presenter.loadDataForSettings()
+                return@setOnMenuItemClickListener true
+            }
+            return@setOnMenuItemClickListener false
+        }
+        sourceImageView.setOnClickListener { presenter.loadTokensForSourceSelection() }
+        destinationImageView.setOnClickListener { presenter.loadTokensForDestinationSelection() }
+        destinationTextView.setOnClickListener { presenter.loadTokensForDestinationSelection() }
+        availableTextView.setOnClickListener { presenter.calculateAvailableAmount() }
+        maxTextView.setOnClickListener { presenter.calculateAvailableAmount() }
+
+        setupAmountListener()
+
+        exchangeImageView.setOnClickListener { presenter.reverseTokens() }
```

Comment:
maybe format all setOn*Listener in one style?

Reply:
We can consider it